site/docs/tutorial/android-app.md

```diff
@@ -3,140 +3,149 @@ layout: documentation
+```
 
-### Update the WORKSPACE file
+## Integrate the Android SDK
 
-Bazel needs to run the Android SDK
-[build tools](https://developer.android.com/tools/revisions/build-tools.html)
-and uses the SDK libraries to build the app. This means that you need to add
-some information to your `WORKSPACE` file so that Bazel knows where to find
-them.  Note that this step is not required when you build for other platforms.
-For example, Bazel automatically detects the location of Java, C++ and
-Objective-C compilers from settings in your environment.
+Bazel needs to run the Android SDK [build
+tools](https://developer.android.com/tools/revisions/build-tools.html) to build
+the app. This means that you need to add some information to your `WORKSPACE`
+file so that Bazel knows where to find them.
 
-Add the following lines to your `WORKSPACE` file:
+Add the following line to your `WORKSPACE` file:
 
 ```python
-android_sdk_repository(
-    name = "androidsdk"
-)
+android_sdk_repository(name = "androidsdk")
 ```
 
 This will use the Android SDK referenced by the `ANDROID_HOME` environment
 variable, and automatically detect the highest API level and the latest version
 of build tools installed within that location.
 
-Alternatively, you can explicitly specify the location of the Android
-SDK, the API level, and the version of build tools to use by including the
+You can set the `ANDROID_HOME` variable to the location of the Android SDK. Find
+the path to the installed SDK using Android Studio's [SDK
+Manager](https://developer.android.com/studio/intro/update#sdk-manager).
+
+For example, as the default SDK path is in your home directory for Linux and
+macOS, and `LOCALAPPDATA` for Windows, you can use the following commands to set
+the `ANDROID_HOME` variable:
+
+| Linux, macOS             | `$ export ANDROID_HOME=$HOME/Android/Sdk/`                                                                                                  |
+| Windows (Command Prompt) | `$ setx ANDROID_HOME "%LOCALAPPDATA%\Android\Sdk"`                                                                                          |
+| Windows (PowerShell)     | `$ [System.Environment]::SetEnvironmentVariable('ANDROID_HOME', "$env:LOCALAPPDATA\Android\Sdk", [System.EnvironmentVariableTarget]::User)` |
+
+Note that on Linux and macOS, the command will set the variable for the current
+shell session only. To make it permanent, you will need to set it as an
+environment variable by adding the line into `~/.bashrc` or `~/.bash_profile`.
+For Windows, the above commands will persist the variable across shells.
+
+Alternatively, you can explicitly specify the location of the Android SDK, the
+API level, and the version of build tools to use by including the
 `path`,`api_level`, and `build_tools_version` attributes. You can specify any
-subset of these attributes:
+combination of these attributes, as long as they are present in the SDK:
 
 ```python
 android_sdk_repository(
```

Comment:
I recommend adding an example for Windows too. The key thing to call out is that `path` must use mixed-style ( see `cygpath --mixed`) paths, i.e. Windows path with forward slashes, e.g. `c:/path/to/sdk`.

Reply:
done. added a Windows example.